### PR TITLE
Update manage-addins-in-the-admin-center.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated explicit admin roles needed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8638

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ description: "Learn about using Centralized add-ins to deploy add-ins to users a
 
 Office Add-ins help you personalize your documents and streamline the way you access information on the web. See [Start using your Office Add-in](https://support.microsoft.com/office/82e665c4-6700-4b56-a3f3-ef5441996862). 
 
-After an admin deploys add-ins for users in an organization, the admin can turn add-ins off or on, edit, delete, and manage access to the add-ins.
+After a global or exchange admin deploys add-ins for users in an organization, they can turn add-ins off or on, edit, delete, and manage access to the add-ins.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md:32]:
```suggestion
After a Global or Exchange admin deploys add-ins for users in an organization, they can turn add-ins off or on, edit, delete, and manage access to the add-ins.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**